src/streetrace/commands/command_executor.py

```diff
@@ -141,7 +141,7 @@ async def execute_async(
 
         logger.info("Executing command: '/%s'", command_name)
         try:
-            await command_instance.execute_async()
+            command_output = await command_instance.execute_async(user_input)
```

Comment:
Should not we limit the length of command_output somehow? Otherwise it could get huge and pollute the prompt (spending tokens).